src/client/game.cpp

```diff
@@ -2973,6 +2981,12 @@ void Game::updateCamera(f32 dtime)
 
 		camera->toggleCameraMode();
 
+#ifdef HAVE_TOUCHSCREENGUI
+		if (g_touchscreengui)
+			g_touchscreengui->setUseCrosshair(m_touch_use_crosshair ||
+					camera->getCameraMode() == CAMERA_MODE_THIRD);
```

Comment:
I would argue this is functionally equivalent to !isNoCrosshairAllowed() since front 3rd person disallows any interaction with the world.

Reply:
That is correct. Thank you for pointing it out!

https://github.com/minetest/minetest/blob/0251b01da66a67ed84f8d27c33aa35f34aaca237/src/client/game.cpp#L3085-L3089

https://github.com/minetest/minetest/blob/0251b01da66a67ed84f8d27c33aa35f34aaca237/src/client/game.cpp#L3990-L3992